plugins/cms-export/src/csv.ts

```diff
@@ -79,9 +79,23 @@ export function getDataForCSV(fields: CollectionField[], items: CollectionItem[]
                     continue
                 }
 
+                case "color": {
+                    if (typeof value !== "string" || isColorStyle(value)) {
+                        columns.push("")
+                        continue
+                    }
+
+                    if (isColorStyle(value)) {
+                        columns.push(value.light)
+                        continue
+                    }
+
+                    columns.push(`${value}`)
+                    continue
```

Comment:
I think it would be simpler without the first early return:

```ts
if (isColorStyle(value)) {
  columns.push(value.light)
  continue
}

columns.push(typeof value === "string" ? value : "")
```

Reply:
And maybe add a `isString` util